src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/CountIf.cs

```diff
@@ -138,16 +138,27 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
                     SuggestDelegationHint(callNode, binding);
                     return false;
                 }
-            }
-
-            // Valiate to see if offline usage hints are applicable.
-            if (binding.DelegationHintProvider?.TryGetWarning(callNode, this, out var warning) ?? false)
-            {
-                SuggestDelegationHint(callNode, binding);
+            }
+
+            // Valiate to see if offline usage hints are applicable.
+            if (binding.DelegationHintProvider?.TryGetWarning(callNode, this, out var warning) ?? false)
+            {
+                SuggestDelegationHint(callNode, binding);
             }
 
             return true;
         }
+
+        private bool ExpressionContainsView(CallNode callNode, TexlBinding binding)
+        {
+            Contracts.AssertValue(callNode);
+            Contracts.AssertValue(binding);
+
+            var viewFinderVisitor = new ViewFinderVisitor(binding);
+            callNode.Accept(viewFinderVisitor);
+
+            return viewFinderVisitor.ContainsView;
+        }
```

Comment:
There are already copies of this function in StatisticalTableFunctions and CountRows.  Should we just expose this once from FunctionWithTableInput?